package.json

```diff
@@ -48,25 +49,25 @@
     "url": "git+https://github.com/bwinton/SnoozeTabs.git"
   },
   "scripts": {
-    "start": "npm run build && npm-run-all --parallel watch run storybook",
-    "start-win": "npm run build && npm run watch",
-    "build": "npm run clean && npm-run-all --parallel copy:assets bundle:js bundle:css",
-    "run": "web-ext run -s dist --firefox=firefoxdeveloperedition",
-    "clean": "rm -rf dist && mkdir dist",
-    "watch": "npm-run-all --parallel watch:js watch:css watch:assets",
-    "copy:assets": "shx cp -r src/manifest.json src/icons dist/ && shx mkdir -p dist/popup && shx cp -r src/popup/*.html dist/popup/",
-    "bundle:js": "npm run lint:js && rollup -c --environment entry:background && rollup -c --environment entry:popup/snooze-content",
+    "build": "npm run clean && npm-run-all --parallel copy:assets bundle:*",
+    "build-storybook": "build-storybook",
     "bundle:css": "npm run lint:sass && shx mkdir -p dist/popup && node-sass src/popup/snooze.scss > dist/popup/snooze.css",
-    "package": "npm run build && web-ext build -s dist -a .",
-    "lint": "npm-run-all --parallel lint:js lint:sass",
+    "bundle:js": "npm run lint:js && rollup -c --environment entry:background && rollup -c --environment entry:popup/snooze-content",
+    "clean": "shx rm -rf dist && mkdir -p dist/popup",
```

Comment:
This needs to call `shx mkdir` instead of `mkdir`, because cygwin's `mkdir` doesn't support the `-p` option.  😭 
(And while we're at it, we don't need the `shx` in front of the `rm -rf`…)